acceptance-tests/src/test/java/org/hyperledger/besu/tests/acceptance/crosschain/viewtxcall/CrosschainCall.java

```diff
@@ -422,7 +422,11 @@ public void doCCTxTxViewCall() throws Exception {
   }
 
   /*
-   * Check that the transaction on a locked contract should fail.
+   * Same as the above test, but the Bar2Ctrt is deployed on chain 1.
+   * Currently this test does not work, because of the nonce issue. First, chain1 creates a crosschain transaction.
+   * The leaf level subordinate transaction is also issued from chain1 later. Because there is only one sender
+   * account per chain, we will have the transaction nonce the same for the both of the above transactions, which
+   * is a problem. We need to fix this later. TODO
```

Comment:
AFAIK, IDEs recognize the text after the TODO to be the actual task. I would recommend putting the TODO keyword at the very beginning.

Reply:
Thanks for that. I already merged it to the master. I am planning to raise a pull request soon. I will include this revision in that.